src/core/components/libp2p.js

```diff
@@ -44,7 +44,7 @@ module.exports = function libp2p (self) {
                 }
               },
               EXPERIMENTAL: {
-                dht: get(opts.options, 'EXPERIMENTAL.dht', false),
+                dht: true,
```

Comment:
It's fine to default to false to get this PR merged so that other folks can try it out